$NetBSD: patch-aa,v 1.1 2009/04/07 21:06:54 seb Exp $

require_gem is deprecated since RubyGems 0.9.0 and
"removed" since RubyGems 1.0.0

--- bin/minitar.orig	2009-04-05 00:40:32.000000000 +0000
+++ bin/minitar
@@ -16,7 +16,7 @@
   # 2) Try to load Archive::Tar::Minitar from $LOAD_PATH.
 begin
   require 'rubygems'
-  require_gem 'archive-tar-minitar', '= 0.5.2'
+  gem 'archive-tar-minitar', '= 0.5.2'
 rescue LoadError
   nil
 end
